script.js

```diff
@@ -15,30 +20,30 @@ window.onload = async function init() {
 
 onClickButton = async () => {
   if (valueInput.length > 0) {
-    allTasks.push({
-      text: valueInput,
-      isCheck: false,
-    });
-    const response = await fetch('http://localhost:8000/createTask', {
-      method: 'POST',
+    pushTasks();
+  } else {
+    alert('Input something!')
+  }
+}
+
+const pushTasks = async () => {
+  const response = await fetch('http://localhost:8000/createTask', {
+    method: 'POST',
     headers: {
       'Content-Type': 'application/json;charset=utf-8',
       'Access-Control-Allow-Origin': '*'
     },
     body: JSON.stringify({
       text: valueInput,
-      isCheck: false,
-    })
+      isCheck: false
+    }),
   });
-  let result = await response.json();
-  allTasks = result.data;
-    valueInput = '';
-    input.value = '';
-    localStorage.setItem('tasks', JSON.stringify(allTasks));
+  const resultPOST = await response.json().then((response) => {
+    allTasks.push(response);
```

Comment:
```
.then((response) => {
    allTasks.push(response);
```
Not sure if this entry is correct.
You can just return the updated array:
`allTasks = result.data;`